### PR TITLE
Add `repository.url` and `repository.provider` labels

### PR DESCRIPTION
**Pull request by actualwitch**
Resolves https://github.com/autometrics-dev/autometrics-py/issues/96

### CODE REVIEW

**Review by flenter — COMMENTED** [review]
Looks good, but can you also update the changelog?

File: README.md [hunk 1]
```diff
@@ -202,17 +202,21 @@ Autometrics makes it easy to identify if a specific version or commit introduced
 >
 > autometrics-py will track support for build_info using the OpenTelemetry tracker via [this issue](https://github.com/autometrics-dev/autometrics-py/issues/38)
 
-The library uses a separate metric (`build_info`) to track the version and, optionally, the git commit of your service.
+The library uses a separate metric (`build_info`) to track the version and git metadata of your code - repository url, provider name, commit and branch.
 
-It then writes queries that group metrics by the `version`, `commit` and `branch` labels so you can spot correlations between code changes and potential issues.
+It then writes queries that group metrics by the set labels so you can spot correlations between code changes and potential issues.
```
Comment (brettimus) [inline README.md:207]: reword: "It then writes queries that group metrics by these labels, so you can spot correlations between code changes and potential issues."

Comment (brettimus) [reply README.md:207]: holdup, i forgot i can suggest a change directly in github, gonna do that

Comment (brettimus) [inline README.md:207]:
```suggestion
It then writes queries that group metrics by these metadata, so you can spot correlations between code changes and potential issues.
```

File: README.md [hunk 2]
```diff
@@ -202,17 +202,21 @@ Autometrics makes it easy to identify if a specific version or commit introduced
...
-| `branch`  | `AUTOMETRICS_BRANCH` or `BRANCH_NAME` | `""`          |
+| Label                 | Run-Time Environment Variables        | Default value |
+| --------------------- | ------------------------------------- | ------------- |
+| `version`             | `AUTOMETRICS_VERSION`                 | `""`          |
+| `commit`              | `AUTOMETRICS_COMMIT` or `COMMIT_SHA`  | `""`          |
+| `branch`              | `AUTOMETRICS_BRANCH` or `BRANCH_NAME` | `""`          |
+| `repository_url`      | `AUTOMETRICS_REPOSITORY_URL`          | `""`\*        |
+| `repository_provider` | `AUTOMETRICS_REPOSITORY_PROVIDER`     | `""`\*        |
+
+\* Autometrics will attempt to automagically infer these values from git config inside working directory. To disable this behavior, explicitly set the corresponding setting or environment variable to `""`.
```
Comment (brettimus) [inline README.md:219]:
```suggestion
\* Autometrics will attempt to automagically infer these values from the git config inside your working directory. To disable this behavior, explicitly set the corresponding setting or environment variable to `""`.
```

File: src/autometrics/tracker/test_concurrency.py [hunk 3]
```diff
@@ -38,29 +38,23 @@ async def test_concurrency_tracking_prometheus(monkeypatch):
...
-# NOTE - Gauges are not supported by the OTEL collector (yet)
-#        https://github.com/open-telemetry/opentelemetry-python/pull/3306
-#
-# @pytest.mark.asyncio
-# async def test_concurrency_tracking_opentelemetry(monkeypatch):
-#     # HACK - We need to set the tracker explicitly here, instead of using `init_tracker`
-#     #        because the library was already initialized with the OpenTelemetry tracker
-#     set_tracker(TrackerType.OPENTELEMETRY)
+@pytest.mark.asyncio
+async def test_concurrency_tracking_opentelemetry(monkeypatch):
```
Comment (brettimus) [inline src/autometrics/tracker/test_concurrency.py:42]: nice!! forgot we could uncomment this test after the otel update

**Review by brettimus — APPROVED** [review]

File: src/autometrics/tracker/test_concurrency.py (hunk 3 above)
Comment (actualwitch) [reply src/autometrics/tracker/test_concurrency.py:38]: yeah i went to update the data there and noticed we can use those now 🎉

**Pull request merged**